cmd/flags.go

```diff
@@ -163,6 +167,22 @@ func fileFlag(v *viper.Viper, cmd *cobra.Command) *cobra.Command {
 	return cmd
 }
 
+func testnetFlag(v *viper.Viper, cmd *cobra.Command) *cobra.Command {
+	cmd.Flags().Bool(flagTestnet, false, "fetches testnet data from the chain registry")
+	if err := v.BindPFlag(flagTestnet, cmd.Flags().Lookup(flagTestnet)); err != nil {
+		panic(err)
+	}
+	return cmd
+}
+
+func forceAddFlag(v *viper.Viper, cmd *cobra.Command) *cobra.Command {
+	cmd.Flags().Bool(flagForceAdd, false, "adds chain data even if there are no working RPC's")
```

Comment:
it could be more clear and avoid some confusion for users if this read something like, `adds chain data even if there are no working RPC's in the chain registry`